extensions/ql-vscode/src/remote-queries/export-results.ts

```diff
@@ -95,3 +97,32 @@ async function exportResultsToGist(
     }
   }
 }
+
+/**
+ * Converts the results of a remote query to markdown and saves the files locally
+ * in the query directory (i.e. the workspace directory where query results and logs are saved).
+ */
+async function exportResultsToLocalMarkdown(
+  queryHistoryManager: QueryHistoryManager,
+  queryHistoryItem: QueryHistoryInfo,
```

Comment:
minor: I see you are passing in two parameters here, `queryHistoryManager` and `queryHistoryItem`, only being used to get the `basePath`. It might be simpler to get the `basePath` outside of this function and pass it in as a parameter.

It cuts down on the # of parameters to this function by one, avoids an import